scripts/tiledb-py/update-recipe.py

```diff
@@ -63,8 +66,20 @@
     yaml.dump(updated, f)
 
 # (Temporary) Update build scripts for scikit-build-core
+remove_deprecations_value = "ON" if day == "Monday" else "OFF"
```

Comment:
You can use `datetime.datetime.today().weekday() == 0` here and avoid the need to shell out.